### DISCUSSION TITLE
Non-existent URL in notification emails for Discussions

### DISCUSSION THREAD

**Discussion by ilg-ul** [community/community#30914] [Other Feature Feedback, Questions, & Ideas]
In the messages I receive as notifications to Discussions, the link at the end of the message points to a non-existent page, in this case it points to:

- <https://github.com/orgs/xpack-dev-tools/discussions/23>

instead of:

- <https://github.com/xpack-dev-tools/arm-none-eabi-gcc-xpack/discussions/23>

The full message is attached:

[[xPack 3rd Party Development Tools] Was libc built to force exceptions_unwind always on? (Discussion #23).eml.zip](https://github.com/community/community/files/9433104/xPack.3rd.Party.Development.Tools.Was.libc.built.to.force.exceptions_unwind.always.on.Discussion.23.eml.zip)

**Comment by amyegan** [comment]
I'm seeing the same problem, and have been since last week. It started for me after incidents with [API requests](https://www.githubstatus.com/incidents/mqmzg413z93k) and [Codespaces](https://www.githubstatus.com/incidents/fhx2r88mzbyv). I thought it might be related and hoped a fix would roll out shortly, but it's still happening ☹️ 

Seems like this isn't limited notifications. The "instead of" link posted above is _also_ redirecting to the broken org level link. 

Happy to help if there's anything I can do or any more information I can provide 🙏 lmk
